inventory/packages/apt_deb.go

```diff
@@ -224,3 +227,15 @@ func InstalledDebPackages() ([]PkgInfo, error) {
 	}
 	return parseInstalledDebpackages(out), nil
 }
+
+// DpkgInstall installs a deb package.
+func DpkgInstall(path string) error {
+	args := append(dpkgInstallArgs, path)
+	out, err := run(exec.Command(dpkg, args...))
+	var msg string
+	for _, s := range strings.Split(string(out), "\n") {
+		msg += fmt.Sprintf(" %s\n", s)
```

Comment:
you're just.. adding a space to the beginning of every line? i don't know why but if so why not replace `'\n'` with `'\n '`

Reply:
This is just for debug log readability. I want to go back and fix all these functions debug logging in the future and maybe get rid of this (right now this is just what all the other functions do).